treescript/src/treescript/l10n.py

```diff
@@ -77,26 +81,82 @@ def build_platform_dict(bump_config, repo_path):
     return platform_dict
 
 
+# get_latest_revision {{{1
+async def get_latest_revision(locale, url):
+    """Download the hg pushlog for the latest locale revision.
+
+    Args:
+        locale (str): the locale to query
+        url (str): the [templatized] pushlog url
+
+    Returns:
+        tuple (locale, revision)
+
+    """
+    url = url % {"locale": locale}
```

Comment:
nit: this should probably use `.format`

Reply:
Hm. I tried this, and it went back to using `default` for every revision. I haven't dug into why. I also have some vague memory there may be a `format` elsewhere that the `%()s` strings avoid.

Tl;dr, I'm probably going to go back to %s because that was working =\